### DISCUSSION TITLE
Add Audio to AR  mode

### DISCUSSION THREAD

**Discussion by ramzidaher** [google/model-viewer#4342] [Q&A]
I'm trying to add audio/sound into the model when it goes into AR mode, I have used src="" and sound=. It works perfectly on Android but on IOS no sound comes out. Can someone help me, that the code I have currently, is it right? 

Code:
    <div id="card">
      <model-viewer
      style="height: 50vh;"
      src="https://cdn.glitch.me/71788b14-e842-4d75-89f4-beae18e6a1d6/unstable_robot.glb?sound=https%3A%2F%2Fcdn.glitch.global%2F71788b14-e842-4d75-89f4-beae18e6a1d6%2FttsMP3.com_VoiceText_2023-7-4_19_47_54.mp3%3F"
      ios-src="https://cdn.glitch.me/71788b14-e842-4d75-89f4-beae18e6a1d6/699dfca9-71ce-4268-8e0a-08280d6da27b.usdz?sound=https%3A%2F%2Fcdn.glitch.global%2F71788b14-e842-4d75-89f4-beae18e6a1d6%2FttsMP3.com_VoiceText_2023-7-4_19_47_54.mp3%3F"
      id="model-viewer" 
      ar ar-modes="webxr scene-viewer quick-look" 
      camera-controls poster="poster.webp" 
      shadow-intensity="1" 
      autoplay animation-name="dance" 
      camera-target="0 0 1"
      sceneViewer: true
      @ar-status="${e => handleARStatus(e)}"
    >

**Comment by elalish** [comment]
Please don't cross-post: #4347.
